pytest_html/__init__.py

```diff
@@ -0,0 +1,11 @@
+from pkg_resources import get_distribution, DistributionNotFound
+try:
+    dist = get_distribution(__name__)
+except DistributionNotFound:
+    # package is not installed
+    pass
```

Comment:
Note that we'd raise an exception when attempting to access `__version__` or `__pypi_url__`.